pyanaconda/modules/network/firewall/firewall.py

```diff
@@ -203,3 +204,18 @@ def set_disabled_services(self, disabled_services):
         self._disabled_services = list(disabled_services)
         self.disabled_services_changed.emit()
         log.debug("Services that will be explicitly disabled on the firewall: %s", self._disabled_services)
+
+    def install_with_task(self, sysroot):
+        """Return the installation tasks of this module.
+
+        :param str sysroot: a path to the root of the installed system
+        :returns: list of object paths of installation tasks
+        """
+        firewall_configuration_task = ConfigureFirewallTask(sysroot=sysroot,
```

Comment:
Please, start the list of arguments on a new line and fix the indentation.

Reply:
Should be fixed now.